docs/modules/ROOT/pages/object-storage/create.adoc

```diff
@@ -2,9 +2,7 @@
 
 AppCat allows DevOps engineers to quickly provision S3-compatible Object Storage Buckets in various environments.
 
-IMPORTANT: In its first release, AppCat Object Storage buckets are only available on the https://portal.appuio.cloud/zones/cloudscale-lpg-2[cloudscale.ch LPG 2 region] of APPUiO Cloud.
```

Comment:
This statement is currently still true... Until we finish up the billing... So probably another sprint.

Reply:
Yes, but I feel it's on the wrong page, now that we have an availability overview in index.adoc...

Reply:
Should we remove it from the availability page then, until it's really there?

Reply:
Done in b78a59a08222ebfbfbd723c4d8c1f3a0615db555